### DISCUSSION TITLE
New store adding

### DISCUSSION THREAD

**Discussion by Murkott** [jef/streetmerchant#1662] [Help]
Hi! 
If I want to add store manually - do I need to insert links manually or bot will get them itself?

**Comment by wofnull** [comment]
nope, you need to create a config file like the ones in src/store/model ( *.ts file ) and set them up to search for the correct contents, further more you need to ad the new config to the main config of the script to bring this to work.

Or you can open a feature request under issues , and ask that the store gets added and somebody will take the time to add the new store in an upcoming release.
